src/components/typography/typography.component.js

```diff
@@ -166,10 +171,14 @@ const Typography = styled.span.attrs(
     line-height: ${lineHeight};
     margin: ${defaultMargin};
     padding: 0;
+    white-space: ${truncate ? "nowrap" : whiteSpace};
+    word-wrap: ${wordWrap};
+    text-overflow: ${truncate ? "ellipsis" : textOverflow};
+    overflow: ${truncate ? "hidden" : overflow};
     ${variant === "sup" && "vertical-align: super;"}
     ${variant === "sub" && "vertical-align: sub;"}
-${display && `display: ${display};`}
-${listStyleType && `list-style-type: ${listStyleType};`}
+            ${display && `display: ${display};`}
```

Comment:
Is this space correct? 🤔

Reply:
Absolutely not 😆 I think my VS Code has auto-formatted this when I've saved it.